stdlib/tarfile.pyi

```diff
@@ -632,6 +631,16 @@ class TarInfo:
     gname: str
     pax_headers: Mapping[str, str]
     def __init__(self, name: str = "") -> None: ...
+    if sys.version_info >= (3, 13):
+        @property
+        @deprecated("Deprecated in Python 3.13; removal scheduled for Python 3.16")
+        def tarfile(self) -> TarFile | None: ...
+        @tarfile.setter
+        @deprecated("Deprecated in Python 3.13; removal scheduled for Python 3.16")
+        def tarfile(self) -> None: ...
```

Comment:
This setter has the wrong signature. It should take a `TarFile | None` argument. I wonder why this passed CI.

Reply:
i'm really sorry for that. my fix: #13904